.travis.yml

```diff
@@ -19,6 +19,7 @@ addons:
     - python-scipy
     - python-matplotlib
     - python-sphinx
+    - python-six
     - texlive-latex-recommended
```

Comment:
It seems that the used version for 'python-six' is 1.1.0-2 wheras required one is at least 1.4
Otherwise I think this should be in a separate PR